exercises/nth-prime/NthPrimeTest.m

```diff
@@ -0,0 +1,53 @@
+#import <XCTest/XCTest.h>
+
+#if __has_include("BobExample.h")
+# import "NthPrimeExample.h"
+#else
+# import "NthPrime.h"
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NthPrimeTest : XCTestCase
+
+@end
+
+@implementation NthPrimeTest
+
+- (void)testFirstPrime {
+	int primeNo = 1;
```

Comment:
What do you think about changing all of these to `primeNum`?

Reply:
I meant it as short for prime number all along, but I do agree that `primeNum` is better.